### PR TITLE
ENGDOCS-1405

### PR DESCRIPTION
**Pull request by aevesdocker**
Troubleshoot topic in DD improvements:
- Style guide alignment
- Adds missing commands for Windows users
- General tidy

### CODE REVIEW

File: desktop/troubleshoot/overview.md [hunk 1]
```diff
@@ -102,87 +98,91 @@ $ /opt/docker-desktop/bin/com.docker.diagnose
...
+  ```console
+  $ <tool location> gather -upload
+  ```
 
-After the diagnostics have finished, the terminal displays your diagnostics ID. The diagnostics ID is
-composed of your user ID and a timestamp. Ensure you provide the full diagnostics ID, and not just the user ID.
+  After the diagnostics have finished, the terminal displays your diagnostics ID. The diagnostics ID is composed of your user ID and a timestamp. Ensure you provide the full diagnostics ID, and not just the user ID.
 
-To view the contents of the diagnostic file, run:
+  To view the contents of the diagnostic file, run:
```
Comment (aevesdocker) [inline desktop/troubleshoot/overview.md:109]: Todo: check what the command is for windows

File: desktop/troubleshoot/overview.md [hunk 2]
```diff
@@ -102,87 +98,91 @@ $ /opt/docker-desktop/bin/com.docker.diagnose
...
-You can then create an issue on GitHub:
-- [For Linux](https://github.com/docker/desktop-linux/issues){:target="_blank" rel="noopener" class="_"}
-- [For Mac](https://github.com/docker/for-mac/issues){:target="_blank" rel="noopener" class="_"}
-- [For Windows](https://github.com/docker/for-win/issues){:target="_blank" rel="noopener" class="_"}
+  You can then create an issue on GitHub:
+   - [For Linux](https://github.com/docker/desktop-linux/issues){:target="_blank" rel="noopener" class="_"}
+   - [For Mac](https://github.com/docker/for-mac/issues){:target="_blank" rel="noopener" class="_"}
+   - [For Windows](https://github.com/docker/for-win/issues){:target="_blank" rel="noopener" class="_"}
 
 ## Check the logs
```
Comment (aevesdocker) [inline desktop/troubleshoot/overview.md:187]: Todo: find out what the equivalent is for windows

File: desktop/troubleshoot/known-issues.md [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
 ---
 <ul class="nav nav-tabs">
-  <li class="active"><a data-toggle="tab" data-target="#tab3">Known issues for Mac with Intel chip</a></li>
-  <li><a data-toggle="tab" data-target="#tab4">Known issues for Mac with Apple silicon</a></li>
+  <li class="active"><a data-toggle="tab" data-target="#tab2">For all platforms</a></li>
+  <li><a data-toggle="tab" data-target="#tab3">For Mac with Intel chip</a></li>
+  <li><a data-toggle="tab" data-target="#tab4">For Mac with Apple silicon</a></li>
 </ul>
 <div class="tab-content">
+<div id="tab2" class="tab-pane fade" markdown="1">
```
Comment (dvdksn) [inline desktop/troubleshoot/known-issues.md:12]: I think this tab body needs the `in active` classes, and it should be removed from the mac with intel tab.

```suggestion
<div id="tab2" class="tab-pane fade in active" markdown="1">
```

File: desktop/troubleshoot/known-issues.md [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
+<br>
+* IPv6 is not yet supported on Docker Desktop.
+<hr>
+</div> 
 <div id="tab3" class="tab-pane fade in active" markdown="1">
```
Comment (dvdksn) [inline desktop/troubleshoot/known-issues.md:17]:
```suggestion
<div id="tab3" class="tab-pane fade" markdown="1">
```

**Comment by aevesdocker** [comment]
Thanks for catching that @dvdksn ⭐

**Pull request merged**